arch/sim/src/sim/up_hostfs.c

```diff
@@ -432,32 +432,35 @@ int host_closedir(void *dirp)
 }
 
 /****************************************************************************
- * Name: host_statfs
+ * Name: host_statvfs
  ****************************************************************************/
 
-int host_statfs(const char *path, struct nuttx_statfs_s *buf)
+int host_statvfs(const char *path, struct nuttx_statvfs_s *buf)
 {
   int            ret;
   struct statvfs hostbuf;
 
-  /* Call the host's statfs routine */
+  /* Call the host's statvfs routine */
 
   ret = statvfs(path, &hostbuf);
   if (ret < 0)
     {
       ret = -errno;
     }
 
-  /* Map the struct statfs value */
+  /* Map the struct statvfs value */
 
-  buf->f_type    = 0; /* hostfs overwrites f_type anyway */
-  buf->f_namelen = hostbuf.f_namemax;
+  buf->f_fsid    = 0; /* hostfs overwrites f_fsid anyway */
+  buf->f_flag    = hostbuf.f_flag;
```

Comment:
f_flags needs host->nuttx conversion. maybe it's safer to use 0 for now.

Reply:
Done.